readthedocs/api/v2/views/integrations.py

```diff
@@ -271,6 +274,10 @@ def handle_webhook(self):
                 raise ParseError('Parameter "ref" is required')
         if event in (GITHUB_CREATE, GITHUB_DELETE):
             return self.sync_versions(self.project)
+
+        if event == GITHUB_PULL_REQUEST:
+            return self.sync_versions(self.project)
```

Comment:
This might be the place where we want to trigger a build after syncing. I'm not 100% sure though.

Reply:
We might need to think a bit more about how we're doing version syncing actually. I think maybe in this case we can just create the version via the incoming PR and trigger the build. That way we don't need to clone the repo 2x.